docs/docs-content/clusters/pcg/deploy-app-pcg.md

```diff
@@ -0,0 +1,641 @@
+
+- **outputs.tf** - Output variables that display information to the terminal once the resources are created. For
+  example, upon cluster deployment, the location of the SSH key and the command to SSH into your cluster nodes are
+  displayed in the terminal.
+
+- **provider.tf** - Terraform providers configuration.
+
+- **ssh-key.tf** - Resources to generate an SSH key pair in case the user does not provide one.
+
+- **ippool.tf** - This file contains the resources required for static IP placement deployments. You will note that the
+  resource `spectrocloud_privatecloudgateway_ippool` is currently commented out as this tutorial uses dynamic IP
+  placement.
+
+- **terraform.tfvars** - Variable definitions required for the deployment. Open the **terraform.tfvars** file in the
+  editor of your choice and provide the values for the variables according to the table below.
+
+  | Variable Name        | Description                                                                                                 | Required |
+  | -------------------- | ----------------------------------------------------------------------------------------------------------- | -------- |
+  | `metallb_ip`         | Provide a valid IP address or IP range from your VMware environment to be assigned to your load balancer.   | Yes      |
+  | `pcg_name`           | Provide the name of the PCG that will be used to deploy the VMware cluster.                                 | Yes      |
+  | `datacenter_name`    | Provide the name of the vSphere datacenter to target when deploying the VMware cluster.                     | Yes      |
+  | `folder_name`        | Provide the name of the vSphere folder to target when deploying the VMware cluster.                         | Yes      |
+  | `search_domain`      | Provide the name of the network search domain of your vSphere environment. For example, `spectrocloud.dev`. | Yes      |
+  | `vsphere_cluster`    | Provide the vSphere compute cluster to use for the VMware cluster deployment.                               | Yes      |
+  | `datastore_name`     | Provide the vSphere datastore name to use for the VMware cluster deployment.                                | Yes      |
+  | `network_name`       | Provide the vSphere network name to use for the VMware cluster deployment.                                  | Yes      |
+  | `resource_pool_name` | Provide the vSphere resource pool to target when deploying the VMware cluster.                              | Yes      |
+  | `ssh_key`            | Provide the path to your public SSH key. If not provided, a new key pair will be created.                   | No       |
+  | `ssh_key_private`    | Provide the path to your private SSH key. If not provided, a new key pair will be created.                  | No       |
+
+When you are ready making the required changes, initialize Terraform with the command presented below.
```

Comment:
We need to make it clearer that the user needs to update the tfvars. We could add that as a paragraph before the current sentence or however you want to make that clear. But keeping that guidance in the bullet for tfvars is not ideal as some will skip reading those bullets.

Reply:
Excellent point!